cli/tests/services.bats

```diff
@@ -890,8 +860,8 @@ EOF
   # as we can get for checking that activation has blocked until the socket
   # exists
   run "$FLOX_BIN" activate -s -- bash <(cat <<'EOF'
-    source "${TESTS_DIR}/services/register_cleanup.sh"
-    "$PROCESS_COMPOSE_BIN" process list
+    sleep 0.1 # ensure services have started
```

Comment:
question: so we are blocking until the socket is created and then still wait to ensure services have started?
Besides we dont check whether the socket file exists either.
I'm not sure this test actually tests what it advertises to test.

Reply:
Fixed in https://github.com/flox/flox/pull/2018/commits/222e5a4a57a65c42314fc0cae3ede143a3e99767